js/events/index.js

```diff
@@ -0,0 +1,123 @@
+import domAdapter from '../core/dom_adapter';
+import eventsEngine from './core/events_engine';
+import KeyboardProcessor from './core/keyboard_processor';
+
+function addNamespace(event, namespace) {
+    return namespace ? eventsEngine.addNamespace(event, namespace) : event;
```

Comment:
Why `addNamespace` method has been left in the `eventsEngine` module? It looks like the namespace adding is the main feature of the current one.